includes/classes/blocks/class-template-parts.php

```
@@ -0,0 +1,267 @@
+				}
+
+				if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
+					error_log( sprintf( 'Created template part: %s (ID: %d)', $slug, $template_part_id ) );
+				}
+			} else {
+				if ( defined( 'WP_DEBUG' ) && WP_DEBUG ) {
+					error_log( sprintf( 'Error creating template part %s: %s', $slug, $template_part_id->get_error_message() ) );
+				}
+			}
+		}
+	}
```

PHPMD (2.15.0)

The method maybe_create_template_part() has a Cyclomatic Complexity of 15. The configured cyclomatic complexity threshold is 10. (undefined)

(CyclomaticComplexity)